package-lock.json

```diff
@@ -1650,24 +1650,24 @@
 			"integrity": "sha512-TlQiXt/vS5ZwY0V3salvlyQzIzMGZEyw9inmJA25A8heL2kBVENbToiEc64R6ETNf5YHa2lwnc2I7iNHP9SqeQ=="
 		},
 		"@floating-ui/core": {
-			"version": "1.2.1",
-			"resolved": "https://registry.npmjs.org/@floating-ui/core/-/core-1.2.1.tgz",
-			"integrity": "sha512-LSqwPZkK3rYfD7GKoIeExXOyYx6Q1O4iqZWwIehDNuv3Dv425FIAE8PRwtAx1imEolFTHgBEcoFHm9MDnYgPCg=="
+			"version": "1.0.1",
```

Comment:
`@floating-ui/core` and `@floating-ui/dom` are deps of the main `@floating-ui/react-dom` from `@wordpress/components`.

But because of `"@floating-ui/dom": "^1.0.0"` these deps resolve to `1.2.1` version. These versions make the editor buggy in Popover issues (see the [GB PR](https://github.com/WordPress/gutenberg/pull/48402) and what issues it solves).

What I did here is:
1. removed the node_modules
2. updated manually the deps of the dep - copied from GB
3. npm install

With these versions as well, there are no Popover issues.

This doesn't feel right though, but I'm not sure what is the best way forward.. @gziolo might have some thoughts?

Reply:
That was it, yes 😆

Reply:
Yes, think of it like you would uninstall old dependencies and install them with forced specific versions. I often remove such deps manually from the lock file and then install them in the project with a pinned version. Once the lock file contains the version I want, I remove them from the `package.json` and run `npm I` again which should confirm no more changes get applied.